package.json

```diff
@@ -36,6 +36,7 @@
     "react-redux": "^7.2.3",
```

Comment:
Plz, pay attention to that the naming pattern as `[FE-3442] Brand item share`. We need to follow it because Jira understand the title and bind PR with tickets